src/View/Helper/FormHelper.php

```diff
@@ -1930,6 +1936,8 @@ public function select($fieldName, $options = [], array $attributes = [])
             return $this->multiCheckbox($fieldName, $options, $attributes);
         }
 
+        unset($attributes['label']);
```

Comment:
That is required to stop `label` getting to `SelectWidget` but still to `MultiCheckWidget`